pkg/api/v1/types.go

```diff
@@ -1413,6 +1433,14 @@ type Container struct {
 	// Cannot be updated.
 	// +optional
 	Env []EnvVar `json:"env,omitempty" patchStrategy:"merge" patchMergeKey:"name" protobuf:"bytes,7,rep,name=env"`
```

Comment:
A couple things:

1.  Put `EnvFrom` before `Env` in this type
2.  Godoc for `Env` should incorporate info about overriding things set from `EnvFrom`